src/components/TodoItem.jsx

```diff
@@ -0,0 +1,38 @@
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react';
+import TodoButton from './TodoButton';
+
+// Todo 항목의 기본 스타일 정의
+const itemStyle = css`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px 0;
+  border-bottom: 1px solid rgba(255, 255, 255, 0.5);
+  cursor: pointer;
+  color: #fff;
+`;
+
+// 완료된 Todo 항목 스타일 정의
+const completedStyle = css`
+  text-decoration: line-through; // 완료된 항목을 취소선으로 표시
+  color: #aaa;
+`;
+
+// 개별 Todo 항목을 나타내는 컴포넌트
+function TodoItem({ todo, index, deleteTodo, toggleTodo }) {
+  return (
+    <div css={[itemStyle, todo.completed ? completedStyle : null]}>
```

Comment:
이 코드와 같은 경우에서도 isCompleted 를 사용하면, "완료 시에는 completeStyle을 사용하고 아직 완료하지 않았다면 아무런 스타일을 적용하지 않는구나" 라고 명확하게 이해하기가 쉬워집니다!

Reply:
조건이 false일 때 null을 리턴하기보다는 `&&` 연산자를 사용해 볼 수 있을 것 같아요~
```suggestion
    <div css={[itemStyle, todo.completed && completedStyle]}>
```
[참고자료 - 단축 평가 논리 계산법](https://learnjs.vlpt.us/useful/03-short-circuiting.html)